proto/README.md

```diff
@@ -118,3 +118,21 @@ code to handle all events for a single resource, without creating
 unnecessary churn when only an unrelated resource type needs
 a new version. (If your code only consumes `bucket.v1` events, the
 introduction of `object.v2` events should not affect you.)
+
+## Unexpected fields
+
+CloudEvent providers may include additional data fields beyond those
+documented here. In particular:
+
+- Some events may include fields with legacy names such as
```

Comment:
Define `legacy names`.

I believe you're calling out that the field uses snake_case instead of camelCase for `message_id` here.

Reply:
I mean names used in the past, that aren't used now.

While the *specific* example we're aware of is using snake_case instead of camelCase, I'm trying to be general. I wouldn't be entirely surprised if we had a similar situation in the future in slightly different ways, and I didn't want to have to rewrite it at that point.